src/Constants/constants.py

```diff
@@ -0,0 +1,7 @@
+ADD_FOODS_SUCCESS_MESSAGE = "Successfully added all foods: {}"
```

Comment:
Could this file be named something else (like "messages.py" or something) since there's already a "constants.py"?

Reply:
Done!